invenio_base/wsgi.py

```diff
@@ -10,8 +10,8 @@
 
 from __future__ import absolute_import, print_function
 
-from werkzeug.contrib.fixers import ProxyFix
-from werkzeug.wsgi import DispatcherMiddleware
+from werkzeug.middleware.dispatcher import DispatcherMiddleware
```

Comment:
I think this will fail with Werkzeug 0.14. Can you try to add a dependency in setup.py with ``werkzeug>=0.14``. This will force requirements-builder to install an earlier version of werkzeug.

Reply:
You are right, for it to work I had to add `werkzeug>=0.15` Is it ok or did we still support werkzeug 0.14?

Reply:
Ok, now supports both versions